requirements/requirements.server.txt

```diff
@@ -4,7 +4,9 @@
 #
 #    pip-compile requirements.server.in
 #
-aiohttp==3.9.5
+aiohappyeyeballs==2.3.5
```

Comment:
A bit odd that it includes the secondary dependency of a secondary dependency, but it's not harmful